velox/functions/sparksql/DecimalUtil.h

```diff
@@ -109,6 +109,21 @@ class DecimalUtil {
     return value;
   }
 
+  template <typename A, typename B>
+  inline static int32_t
+  minLeadingZeros(const A& a, const B& b, uint8_t aScale, uint8_t bScale) {
+    int32_t aLeadingZeros = bits::countLeadingZeros(absValue<A>(a));
```

Comment:
nit: absValue(a) and absValue(b)

Reply:
It generates below error if removing the template type.
> couldn’t deduce template parameter ‘T’
  118 |     int32_t bLeadingZeros = bits::countLeadingZeros(absValue(b))